File: Ionic/VisitorBook/App/dailyVisitors/src/app/models/office.ts
export class Office  {
    officeId: number;
    empOfficeId:number;
    officeName:string;
    countryId: number;
    address: string;
    city:string;
    state:string;
    zipCode:string;
    oUName:string;
    active:boolean;
    countryName:string;


    constructor();   
    constructor(object?: any) {
        this.officeId = object && object.officeId || null;
        this.empOfficeId = object && object.empOfficeId || "";
        this.officeName = object && object.officeName || "";
        this.countryId = object && object.countryId || "";
        this.address = object && object.address || null;
        this.city = object && object.city || null;
        this.state = object && object.state || null;
        this.zipCode = object && object.zipCode || null;
        this.oUName = object && object.oUName || "";
        this.active = object && object.active || "";
        this.countryName = object && object.countryName || "";
    }
    
}

File: Ionic/VisitorBook/DailyVisitors/DAL/Models/Office.cs
using System;
using System.Collections.Generic;

namespace DailyVisitors.DAL.Models
{
    public partial class Office
    {
        public int OfficeId { get; set; }
        public Int16 EmpOfficeId { get; set; }
        public string OfficeName { get; set; }
        public Int16 CountryId { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string ZipCode { get; set; }
        public string OUName { get; set; }
        public bool Active { get; set; }
        public string CountryName { get; set; }
    }
}
